pkg/image/apiserver/apiserver.go

```diff
@@ -116,11 +125,41 @@ func (c *completedConfig) V1RESTStorage() (map[string]rest.Storage, error) {
 }
 
 func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
-	// TODO: allow the system CAs and the local CAs to be joined together.
-	importTransport, err := restclient.TransportFor(&restclient.Config{})
+	cfg := restclient.Config{}
+
+	tlsConfig := &tls.Config{}
+
+	var err error
+	tlsConfig.RootCAs, err = x509.SystemCertPool()
+	if err != nil {
+		return nil, fmt.Errorf("unable to get system cert pool for default transport for image importing: %v", err)
+	}
+	if tlsConfig.RootCAs == nil {
+		tlsConfig.RootCAs = x509.NewCertPool()
+	}
+
+	glog.V(2).Infof("Image import using additional CA path: %s", c.ExtraConfig.AdditionalTrustedCA)
+	if len(c.ExtraConfig.AdditionalTrustedCA) != 0 {
+		if _, err := os.Stat(c.ExtraConfig.AdditionalTrustedCA); !os.IsNotExist(err) {
+			caData, err := ioutil.ReadFile(c.ExtraConfig.AdditionalTrustedCA)
+			if err != nil {
+				return nil, fmt.Errorf("failed to read ca bundle for image importing: %v", err)
+			}
+			tlsConfig.RootCAs.AppendCertsFromPEM(caData)
```

Comment:
AppendCertsFromPEM() could return false.

Reply:
addressed in `david2`